trpc/router/goal.ts

```diff
@@ -895,18 +910,26 @@ export const goal = router({
                 },
             });
 
+            const actualGoal = await prisma.goal.findUnique({ where: { id: newComment.goalId } });
```

Comment:
I'm not sure we need to trigger project update on comment update. Let's remove it.